lib/commands/command-list-all.bash

```diff
@@ -8,7 +8,15 @@ list_all_command() {
   check_if_plugin_exists "$plugin_name"
 
   local versions
-  versions=$(bash "${plugin_path}/bin/list-all")
+  # Capture return code to allow error handling
+  return_code=0 && versions=$(bash "${plugin_path}/bin/list-all" 2>&1) || return_code=$?
+
+  if [[ $return_code -ne 0 ]]; then
+    # Printing all output to allow plugin to handle error formatting
+    printf "Plugin %s's list-all callback script failed with output:\\n" "${plugin_name}"
+    printf "%s\\n" "${versions}"
```

Comment:
I think these prints should go to STDERR instead of STDOUT. Generally only good data should be printed to STDOUT as it is often consumed by other programs.

Reply:
I've pushed the requested change and all looks good.